eldritch/events.py

```diff
@@ -2983,6 +2989,27 @@ def finish_str(self):
     return f"{self.character.name} did not defeat the {self.monster.name}"
 
 
+class PassCombatRound(Event):
+  def __init__(self, combat_round, log_message="Combat Round Passed"):
+    self.combat_round = combat_round
+    self.log_message = log_message
+    self.done = False
+
+  def resolve(self, state):
+    self.combat_round.defeated = True
+    self.done = True
+    return True
+
+  def is_resolved(self) -> bool:
+    return self.done
+
+  def start_str(self):
+    return self.log_message
```

Comment:
I might (a) change the log message to look more like the others (e.g. character.name passed a combat round against a monster.name) and (b) put this in finish_str instead of start_str